### ISSUE TITLE
Ambiguous watchOS instructions in README.md

### ISSUE THREAD

**Issue by SuperKenVery** [dagronf/QRCode#3]
After reading `README.md`, it's easy for me to think that, in order to generate QR codes on watchOS, I must add `QRCode` **and** `QRCode3rdPartyGenerator` to my project, while in fact I should **only** add `QRCode3rdPartyGenerator`, or the project would fail to build.

**Comment by dagronf** [comment]
Hi there,

Apologies for the confusion here. I've just pushed up a new build (tagged 7.0.0) which includes a separate watchOS library target (`QRCodeWatchOS` and `QRCodeWatchOSDynamic`) which will hopefully reduce the confusion here. Admittedly when exploring your comment I stumbled over exactly the same issue!

So, long answer short - you should upgrade to 7.0.0 and link against `QRCodeWatchOS` (much easier to understand!)

* I've also updated the doco to include this change, along with a screenshot of the `Frameworks, Libraries, and Embedded Content` requirement.
* There's also a new bare-bones watchOS example in the `Demos` folder.

Can you take a look and see what you think?

Cheers and thanks for the feedback!

**Comment by SuperKenVery** [comment]
Great job! It's now much easier to understand how to generate QR code on watchOS!

However, I think there are 2 tiny things:
1. Despite its name, `QRCodeWatchOS` also works (perfectly) on iOS/macOS/tvOS. If one wants to keep the same code between iOS/watchOS to generate QR code, this may add confusion to him/her; maybe leading he/her to use different codes on iOS and watchOS. 
2. The `QRCode3rdPartyGenerator` seems to still be there. Maybe consider removing it?

**Comment by dagronf** [comment]
Agreed.

* `QRCodeWatchOS` has been renamed to `QRCodeExternal`
* `QRCodeGenerator_WatchOS` has been renamed to `QRCodeGenerator_External`
* `QRCode3rdPartyGenerator` has been removed completely.

The README.md file has been updated to reflect the changes, also that `QRCodeExternal` can be used for all platforms.

I've updated the version number to 8.0.0 so that anyone who has upgraded to 7.0.0 won't automatically update to the new naming until they want/need to.
